wp-content/themes/goonj-crm/functions.php

```diff
@@ -14,3 +14,12 @@ function goonj_theme_setup() {
     add_editor_style('style.css');
 }
 add_action('after_setup_theme', 'goonj_theme_setup');
+
+# Redirecting to civi dashboard if already logged in
+function redirect_logged_in_user_to_civi_dashboard() {
```

Comment:
All functions in theme should be prefixed with `goonj_` to avoid any conflicts.

```suggestion
function goonj_redirect_logged_in_user_to_civi_dashboard() {
```

Reply:
Perfect. Updated